### DISCUSSION TITLE
Flash Flood Guidance

### DISCUSSION THREAD

**Discussion by lazarus13** [weather-gov/api#396] [Q&A]
The former Flash Flood Guidance text product that used to be provided has been updated to a new grid based image system.  

Is it possible to add either the former text based observations/forecast for FFG or a pre generated FFG image for the gridpoint to the API.

**Comment by StephenClouse** [comment]
Per [NWS SCN 18-80](https://www.weather.gov/media/notification/pdfs/scn18-80disc_county_ffgaab.pdf) the text-based FFG products were terminated on March 31, 2019 and are no longer available.

We don't currently have any definitive plans to integrate the gridded FFG into the API. For now we would recommend fetching the GRIB data from the WPC FTP site at [ftp://ftp.wpc.ncep.noaa.gov/workoff/ffg/](ftp://ftp.wpc.ncep.noaa.gov/workoff/ffg/)

**Comment by scadergit** [comment]
I will reach out to IDPGIS and determine if this data could be ingested into the geoserver. That would be the best option to disseminate the data as images.
